python/paddle/v2/fluid/learning_rate_decay.py

```diff
@@ -123,3 +123,54 @@ def inverse_time_decay(learning_rate,
         div_res = layers.floor(x=div_res)
 
     return learning_rate / (1 + decay_rate * div_res)
+
+
+def polynomial_decay(learning_rate,
+                     global_step,
+                     decay_steps,
+                     end_learning_rate=0.0001,
+                     power=1.0,
+                     cycle=False):
+    """Applies inverse time decay to the initial learning rate.
```

Comment:
This should be `Applies polynomial decay to the initial learning rate.`

Reply:
ok~